scripts/generate-test-anchors

```diff
@@ -0,0 +1,168 @@
+
+Options:
+  --canister-id CANISTER_ID       Canister id of the II canister to generate the test anchors with.
+  --pubkey PATH                   Path to a file containing the DER encoded public key that dfx is using.
+  --network URL                   Optional network parameter. Defaults to local replica.
+  --count NUMBER_OF_ANCHORS       The number of anchors to generate.
+EOF
+}
+
+function help() {
+    cat << EOF
+
+Generates many test anchors on the specified II instance.
+
+NOTE: This requires dfx, hexdump, sed, running II canister with the II_DUMMY_CAPTCHA flavor and a public key file of the dfx identity.
+The public key file can be obtained using
+
+    dfx identity export IDENTITY | openssl ec -out FILE_NAME -pubout -outform der
+
+This works only for identities generated with dfx 0.12.0 or higher. The public key must correspond to the identity that
+is active in dfx when running this script. To see which identity is active run "dfx identity list". To switch to a different
+identity run "dfx identity use IDENTITY". It is recommended to use an identity without encryption as the passphrase has
+to be submitted for every operation.
+EOF
+}
+
+# ARGUMENT PARSING
+
+while [[ $# -gt 0 ]]
+do
+    case $1 in
+        --help)
+            title
+            usage
+            help
+            exit 0
+            ;;
+        --pubkey)
+            pubkey="${2:?missing value for '--pubkey'}"
+            shift; # shift past --pubkey and value
+            shift;
+            ;;
+        --canister-id)
+            canister_id="${2:?missing value for '--canister-id'}"
+            shift; # shift past --canister-id and value
+            shift;
+            ;;
+        --network)
+            network="${2:?missing value for '--network'}"
+            shift; # shift past --network and value
+            shift;
+            ;;
+        --count)
+            count="${2:?missing value for '--count'}"
+            shift; # shift past --count and value
+            shift;
+            if ! [[ $count =~ $number_check ]] ; then
+               echo "error: count \"$count\" is not a number" >&2; exit 1
+            fi
+
+            ;;
+        *)
+            echo "ERROR: unknown argument $1"
+            usage
+            echo
+            echo "Use 'release --help' for more information."
+            exit 1
+            ;;
+    esac
+done
+
+if [ -z "$pubkey" ]
+then
+    echo no public key path
+    usage
+    exit 1
+fi
+
+if [ -z "$canister_id" ]
+then
+    echo no canister id
+    usage
+    exit 1
+fi
+
+if [ -z "$count" ]
+then
+    echo no count
+    usage
+    exit 1
+fi
+
+declare -a network_arg=( )
+
+if [ "$network" ]
+then
+    network_arg+=( "--network" "$network")
+fi
+
+function create_anchor {
+    challenge_key=$(dfx canister "${network_arg[@]}" call "$canister_id" create_challenge --candid src/internet_identity/internet_identity.did -qq | sed -n 's/.*challenge_key[[:space:]]*=[[:space:]]*"\(.*\)".*/\1/p')
```

Comment:
out of curiosity: is there nothing like `jq` for candid, whom we could tell "grab the `challenge_key` field?